test/lib/mix/gear_static_analysis_test.exs

```diff
@@ -0,0 +1,214 @@
+defmodule Mix.Tasks.Compile.GearStaticAnalysisTest do
+  use Croma.TestCase
+
+  @tmp_ex_file_path Path.join(__DIR__, "tmp_ex_file_for_gear_static_analysis_test.ex")
+
+  setup do
+    :meck.expect(Mix.Project, :config, fn ->
+      [{:antikythera_gear, [use_antikythera_internal_modules?: false]} | :meck.passthrough([])]
+    end)
+
+    on_exit(fn ->
+      :meck.unload(Mix.Project)
+      File.rm!(@tmp_ex_file_path)
+    end)
+  end
+
+  # Gear name is `antikythera` in the following tests
+  describe "find_issues_in_file/1" do
+    test "should detect naming module which is not prefixed with the gear name" do
+      File.write!(@tmp_ex_file_path, """
```

Comment:
I guess you can use `@tag :tmp_dir`.
https://hexdocs.pm/ex_unit/1.13.4/ExUnit.Case.html#module-tmp-dir

Reply:
Thank you.
I didn't know this useful feature!
Fixed at https://github.com/access-company/antikythera/pull/264/commits/172666e9024efe45c7587100b99af2d3ef4555fd